client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,103 @@
+
+        </svg>)
+}
+const MessageIcon = () => {
+    return (
+        <svg className="messageIcon hover" width="19" height="13" viewBox="0 0 19 13" fillOpacity="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M0 0V12.5945H19V0H0ZM9.5001 7.77761L1.70706 0.934204H17.2929L9.5001 7.77761ZM6.3957 6.29503L0.934204 11.0911V1.49895L6.3957 6.29503ZM7.10358 6.91676L9.49986 9.02112L11.8961 6.91676L17.2978 11.6603H1.70217L7.10358 6.91676ZM12.6041 6.29526L18.0658 1.49895V11.0911L12.6041 6.29526Z" fill="black" fill-opacity="0.5" />
+        </svg>)
+
+}
+const NotifyIcon = () => {
+    return (
+        <div className="notifyIcon hover"> <svg width="14" height="15" viewBox="0 0 14 15" fillOpacity="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M13.9503 14.3169L12.7484 12.3134C12.1933 11.3887 11.9 10.3295 11.9 9.25151V7.49653C11.9 5.28094 10.4215 3.40698 8.39983 2.80284V1.54631C8.39983 0.774165 7.77191 0.14624 6.99976 0.14624C6.22762 0.14624 5.5997 0.774165 5.5997 1.54631V2.80284C3.57804 3.40698 2.09958 5.28094 2.09958 7.49653V9.25151C2.09958 10.3295 1.80626 11.388 1.25184 12.3127L0.0499042 14.3162C-0.0151984 14.4247 -0.0166093 14.5591 0.0457041 14.669C0.108017 14.7789 0.223522 14.8468 0.349527 14.8468H13.65C13.776 14.8468 13.8922 14.779 13.9545 14.6698C14.0169 14.5605 14.0148 14.4247 13.9503 14.3169Z" fill="black" fill-opacity="0.5" />
+        </svg>
+            <svg width="6" height="3" viewBox="0 0 6 3" fillOpacity="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M0.740234 0.793091C1.13505 1.61773 1.97088 2.19316 2.9446 2.19316C3.91831 2.19316 4.75418 1.61773 5.149 0.793091H0.740234Z" fill="black" fill-opacity="0.5" />
+            </svg>
+        </div>
+    )
+}
+
+const user = {
+    name: "Sofi Dub",
+    image: "https://s3-alpha-sig.figma.com/img/919e/1a5a/da4f250d469108191ad9d4af68b2a639?Expires=1566172800&Signature=Kou41Z8bd8ig~9nLibgCH5gfaOc0K~9Io82-umabjJnomveXbPcqMWfD911bHy6h77reHT6ecNYFHCzmXkQNy3vEF-OzgJYgV875TI2rX~cPt1FaSJC5wCeybEfTrlBlCcdzSFn8iVcP~C8GTx-l6CIjyugGAhvr7xJ-hfAdlf~5Mll0Sy92dSKn8q7OkJdfsMvEEFVQ3rGHn8GGQZg1a60gif0VaQhuVX1gcRgwrsak~cerS1bnDvo93B1lFOIk85wlhY2hPwQrmCtI9A-qaAtbIxmzmxkRpuVUpDrX6Jd4hXpksbd7urSJ91Dg7tv9WzRZvIkLnPXflCfmPw~slw__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA"
+}
+const mock = ["Movies in cinema", "Top movies", "On DVD"];
+
+const Header = ({ userInfo=user, movies = mock, tv = mock, ratings = mock }: IProps) => {
+    return (
+        <div className="header">
+            <div className="title">Pop Corn</div>
+            <button className="headerButtons hover">
```

Comment:
Please use consistent class name format. I'd suggest to switch to hyphens instead of camelCase